packages/api/lib/ingest.js

```diff
@@ -91,14 +65,6 @@ async function reingestGranule({
     asyncOperationId,
   });
 
-  // FUTURE This would ideally not be necessary
-  await _updateGranuleStatus(
```

Comment:
As far as I can tell, these calls are unnecessary. They are just setting the granule status to `running` immediately before the Lambda is called to send these workflow messages to the SQS queue (the Lambda is `process.env.invoke` which is `<prefix>-ScheduleSF`).

First of all, this logic is unnecessary. Because when the workflow message is eventually picked up from the queue and started, the granule should automatically transition to a state of `running`. And if that doesn't happen, then we have a bigger issue.

Second, this logic is actually harmful. By pre-emptively setting the granule status to `running` before it is even queued, we are giving users watching granules in the dashboard a false impression of the state of the granule. The granule may actually fail to be queued or started in subsequent logic, but it would still already have been put in a `running` state.

So I feel pretty strongly that removing this code is the right thing to do.

Reply:
Pasting from a DM with Jonathan:

This came from https://github.com/nasa/cumulus/pull/2415/files#diff-7eb3e889de5c30926d58949a13ff986c5dba528f4e42a9692af27c825e0e86d9L213 

I think the only reason I'm explicitly setting the status there is because the original `Granule.reingest` method that `reingestGranule` replaces did it. I totally agree that the workflow should set the status to running and it shouldn't be necessary but at the time we (I) didn't want to change the "already working" logic aside from moving it wholesale.

My thinking at the time was that this was, at most, redundant. If it's potentially causing issues it's definitely worth removing and seeing if tests break.

Reply:
I did a bit of digging as https://github.com/nasa/cumulus/pull/2415/ made a point to implement this logic.   From a convo with @npauzenga , *that* was due to our agreed approach of not attempting changes like these in phase 2 to scope-limit the work and avoid more complex consequences of the feature branch. 

That, however, doesn't explain the reasoning why the logic was implemented in the first place.

Reply:
In either case, :100: re the original assertion.

Reply:
OK, well even if the logic was kept around to match an existing implementation, at this point I'm inclined to remove it given the possible confusion it could generate for dashboard operators.

Reply:
I tend to agree. 

There's a functional change to this method (demonstrated by the unit changes)  that *now* if the granule provided doesn't exist, this method won't throw.      That's probably a *good* change, two comments:  

1) as we're co-opting this method in our integration tests I took a quick look at the affected specs and it looks ok, commenting in case that triggers something else for @markdboyd  or others reading the review. 

2) The 'doesn't exist' issue should already be covered by `getUniqueGranuleByGranuleId` in the endpoint logic, so that's 💯